test/integration/uds_integration_test.h

```diff
@@ -46,4 +46,31 @@ class UdsIntegrationTest
 protected:
   const bool abstract_namespace_;
 };
+
+class UdsListenerIntegrationTest : public HttpIntegrationTest,
+                                   public testing::TestWithParam<std::tuple<bool>> {
+public:
+  UdsListenerIntegrationTest()
+      : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, Network::Address::IpVersion::v4),
+        abstract_namespace_(std::get<0>(GetParam())) {}
+
+  void initialize() override;
+
+  std::string getAdminSocketName() {
+    return abstract_namespace_
+               ? "@/envoy-tests/uds-integration-test/admin"
```

Comment:
I think we need some variation in the name to avoid collisions between concurrent test runs.

Reply:
Done.